### ISSUE TITLE
Create the Subject details page in React

### ISSUE THREAD

**Issue by Sanyam2000-dot** [ADRE9/bunk-manager-mern#146]
**Deliverables:**
Add the deliverables with their description here.

- [ ]  Link for design is https://www.figma.com/file/8JiFfRS746ot9w7Uh7yq3O/Untitled?node-id=0%3A1
- [ ] Use Material UI
- [ ] Make for both light and dark theme

![Screenshot (251)](https://user-images.githubusercontent.com/53525430/117499153-718eb600-af98-11eb-9663-d04aff156019.png)
![Screenshot (252)](https://user-images.githubusercontent.com/53525430/117499206-866b4980-af98-11eb-83e4-8422418f28d9.png)

**Comment by Preeti2095** [comment]
Can we create these using figma @Sanyam2000-dot ??

**Comment by Preeti2095** [comment]
I would like to work in this issue.

**Comment by Sanyam2000-dot** [comment]
Sure, Go ahead! In case you face any doubt please comment here only. Happy coding :)

**Comment by Preeti2095** [comment]
Okay, Thank you.

**Comment by Preeti2095** [comment]
![Screenshot (73)](https://user-images.githubusercontent.com/70755131/117923204-45b15e80-b311-11eb-9854-e65d57fb28ab.png)
i have done this much. To put the calendar i am planning to do it in card. i am having problem to put that cards for calendar. Please help me in that or any suggestion @Sanyam2000-dot .

**Comment by ADRE9** [comment]
@Preeti2095 for now put the calendar aside and work on the rest of the page. Calendar will need another issue of level three. Just put a blank div for calendar with a gray color and a label as Calendar to specify that a calendar would be integrated there. And make sure you use exact colors.

**Comment by Preeti2095** [comment]
Okay.

**Comment by Preeti2095** [comment]
Hello @Sanyam2000-dot @ADRE9. I would like to make a pr to show what i have done till now so that i can further improve my code and i have few doubts also. So can i make the pr?

**Comment by ADRE9** [comment]
It would be better if you can show us some of the screenshots. And after that you can make a PR so that we can review.@Preeti2095

**Comment by Preeti2095** [comment]
this is what i have done @ADRE9 @Sanyam2000-dot
![Screenshot (74)](https://user-images.githubusercontent.com/70755131/118368070-dc567780-b5be-11eb-9977-bd91743d0571.png)

**Comment by Sanyam2000-dot** [comment]
@Preeti2095  This looks good but try to optimize it more according to the design given to you.

**Comment by Preeti2095** [comment]
yes, i am doing that. One question how can i make the text ( BUNK MANAGER) and the logo(small tick mark sign) on same line?

**Comment by Preeti2095** [comment]
UPDATE:-

![Screenshot (75)](https://user-images.githubusercontent.com/70755131/118774742-c89f5f80-b8a3-11eb-8c3a-c7bf0b2a884c.png)

i want to ask how can i put progress bar like it will be connected to graph, right?

**Comment by ADRE9** [comment]
For now just complete the the design and make sure you are using Material UI for the designs. Make the progress bar using [this](https://material-ui.com/components/progress/#linear-determinate)

**Comment by Preeti2095** [comment]
i think it will be better if i use [this](https://material-ui.com/components/slider/#slider) then progress bar. What do you think @ADRE9?

**Comment by Preeti2095** [comment]
I am waiting for your response @ADRE9 @Sanyam2000-dot.

**Comment by ADRE9** [comment]
> I am waiting for your response @ADRE9 @Sanyam2000-dot.

Go for it. Sorry for not responding. Had some commitments.

**Comment by Preeti2095** [comment]
> > I am waiting for your response @ADRE9 @Sanyam2000-dot.
> 
> Go for it. Sorry for not responding. Had some commitments.

It's alright!

**Comment by Preeti2095** [comment]
Update @ADRE9 @Sanyam2000-dot :-
![Screenshot (77)](https://user-images.githubusercontent.com/70755131/120467514-e39ec300-c3bd-11eb-9f0b-2aa45f0211be.png)

**Comment by Preeti2095** [comment]
It is pretty much done and just left with light mode and make it responsive.
So for light mode shall I create a different page?

**Comment by Preeti2095** [comment]
> It is pretty much done and just left with light mode and make it responsive.
> So for light mode shall I create a different page?

?? @ADRE9 @Sanyam2000-dot

**Comment by ADRE9** [comment]
> It is pretty much done and just left with light mode and make it responsive.
> So for light mode shall I create a different page?

Light mode just needs it to be using theme. You don't need to create another Page. And yes make it responsive.

**Comment by Preeti2095** [comment]
including theme is getting overwhelming kind of thing for me....can you see my code and guide me ahead?? i can't do ahead alone. Will be greatful for your guidance @ADRE9  @Sanyam2000-dot or any other project admins or anyone.

**Comment by ADRE9** [comment]
Hey @Preeti2095 , we are at the moment pretty busy. Glad you are so interested in learning. I'll help you as soon as I get some time.

**Comment by Preeti2095** [comment]
Ok sure.
